en/prev/Content/Topics/Adv_Server_Access/asa-home.htm

```diff
@@ -1,122 +1,10 @@
+<html>
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta charset="utf-8" />
-        <meta name="typeofcontent" content="documentation" />
-        <meta name="audience" content="admin" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-        <meta http-equiv="REFRESH" content="0.0;url=https://help.okta.com/okta_help.htm?type=asa&amp;id=ext_asa_main" /><title>asa-home</title>
-        <link href="../../../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/Components/Tablet.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/Components/Mobile.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Fluid/Stylesheets/foundation.6.2.3.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Fluid/Stylesheets/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Fluid/Stylesheets/Tablet.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Fluid/Stylesheets/Mobile.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../Resources/Stylesheets/okta-main-new-nav.css" rel="stylesheet" type="text/css" />
-        <script src="../../../Resources/Scripts/jquery.min.js" type="text/javascript">
-        </script>
-        <script src="../../../Resources/Scripts/purify.min.js" type="text/javascript" defer="defer">
-        </script>
-        <script src="../../../Resources/Scripts/require.min.js" type="text/javascript">
-        </script>
-        <script src="../../../Resources/Scripts/require.config.js" type="text/javascript" defer="defer">
-        </script>
-        <script src="../../../Resources/Scripts/foundation.6.2.3_custom.js" type="text/javascript">
-        </script>
-        <script src="../../../Resources/Scripts/plugins.min.js" type="text/javascript" defer="defer">
-        </script>
-        <script src="../../../Resources/Scripts/MadCapAll.js" type="text/javascript" defer="defer">
-        </script>
+        <meta name="robots" content="noindex, nofollow" />
+        <meta http-equiv="REFRESH" content="0.0;url=https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/asa-home.htm" />
```

Comment:
Double checking - the previous url had `en/prev` (does this mean it's a preview topic?) but the new one does not - is that okay?